coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/Calcite2TrinoUDFConverter.java

```diff
@@ -279,6 +287,23 @@ public RexNode visitCall(RexCall call) {
       return Optional.empty();
     }
 
+    private Optional<RexNode> visitCast(RexCall call) {
```

Comment:
Could you please add a commend with an example here?

Reply:
Added in revision